homeassistant/helpers/condition.py

```diff
@@ -163,7 +161,18 @@ def if_or_condition(
     return if_or_condition
 
 
-or_from_config = _threaded_factory(async_or_from_config)
+async def async_device_from_config(
```

Comment:
This should not be part of the `condition` helper, instead I think it should be part of `components/device_automation/__init__.py`

Reply:
Moved.